packages/storage/src/indexed-db/index.ts

```diff
@@ -885,4 +885,24 @@ export class IndexedDb implements IndexedDbInterface {
       );
     });
   }
+
+  async getAssetTokenMetadata(
```

Comment:
Don't think this name is accurate. This should be something like: `hasTokenBalance`.

Reply:
nice catch, changed